### ISSUE TITLE
Building gadgets gated issue

### ISSUE THREAD

**Issue by Mdial1** [Iskallia/Vault-public#629]
Building gadgets are 'locked behind research' for others even if someone else (who has it unlocked) crafted them, limiting trading/usefulness as they aren't able to use them.

**Comment by bmemike** [comment]
I believe this is working as intended.

If you allow users to fully use all items from locked research crafted by others, you effectively reduce research costs by a factor of the amount of players on a server.

This greatly undermines the research system balance as designed.

**Comment by Mdial1** [comment]
> I believe this is working as intended.
> 
> 
> 
> If you allow users to fully use all items from locked research crafted by others, you effectively reduce research costs by a factor of the amount of players on a server.
> 
> 
> 
> This greatly undermines the research system balance as designed.

Yes to an extent. But at the same time this isn't the case with other mods. For example, a mekanism crusher can be used by others. And a better example, easy villagers used to be locked and unable to be interacted with unless research was unlocked. However in more recent updates one can interact with them without the research. I feel like this is a similar circumstance

**Comment by Mdial1** [comment]
Another example I just thought of, is the gluttony charm. An OP item that can be used by others. Definitely a consideration and your point of it decreasing the research is valid. But I feel like trading is the point of it being with other people. Just my thoughts
